include/nanogui/common.h

```diff
@@ -494,7 +504,13 @@ extern NANOGUI_EXPORT std::vector<std::pair<int, std::string>>
 
 /// Convenience function for instanting a PNG icon from the application's data segment (via bin2c)
 #define nvgImageIcon(ctx, name) nanogui::__nanogui_get_image(ctx, #name, name##_png, name##_png_size)
+
 /// Helper function used by nvgImageIcon
 extern NANOGUI_EXPORT int __nanogui_get_image(NVGcontext *ctx, const std::string &name, uint8_t *data, uint32_t size);
 
+namespace detail {
+    /// Used for scaling icons (primarily Entypo) in widget draw calls.
+    static constexpr float __icon_scale = 0.77f;
```

Comment:
The new Entypo+ font is presumably bigger.  See annotations in various CPP files for how this is used.

Reply:
Sounds fair. I don't think it needs to have such a cryptic name (``__icon_scale``) and be ``constexpr``. The ``Theme`` class would seem like a more natural home to me.

Reply:
I just realized that ``entypo`` has a CC-BY-SA license, which is a bit more restrictive than the BSD license that is used for the rest of NanoGUI. Unlikely to be an issue to anyone, but it may be worth adding a note somewhere in the docs.

Reply:
Should it go on the README?

Reply:
As for cryptic name, I'm adding it to the theme.  The `__` came from `__nanogui_get_image` above it...but I see now that's because of the `define` above xD